### DISCUSSION TITLE
compareBuildDetails unavailable in release pipelines ?

### DISCUSSION THREAD

**Discussion by CosmicWebCreator** [rfennell/AzurePipelines#936] [Q&A] [answered]
Hi,
First of all, thank you for the plugin ! It works very well and easy to install. However, I was trying to use this in my release pipeline in azure and I can't seem to get the field compareBuildDetails filled out. Is it because it's only available at build pipelines or I am doing something wrong ? I have enabled Stage check...

Any help is very appreciated !

Thank you !

**Comment by rfennell** [comment, marked as answer]
The `compareBuildDetails` parameter is only valid if  `checkstage=true` and you are using either a classic UI based build or a YAML Pipeline

It is not supported in classic UI based releases.

**Reply by rfennell** [reply]
Sorry should have said you need in a classic UI release the equivalent is `compareReleaseDetails`.

See the [wiki](https://github.com/rfennell/AzurePipelines/wiki/GenerateReleaseNotes---Node-based-Cross-Platform-Task) for the complete list

**Reply by CosmicWebCreator** [reply]
Ah HA! Thank you very much for the time you took to answer my question.

**Reply by CosmicWebCreator** [reply]
Oh but I just noticed that it doesn't have a buildNumber on the compareReleaseDetails right ?

**Reply by rfennell** [reply]
No in a Classic UX basd release it will have fields like a `releaseID` and `releasename`. You have to look back into the artefacts to see the builds related to the release
